packages/runtime/src/editor-preview.main.ts

```diff
@@ -1167,7 +1176,10 @@ export const createRoot = (
     dataSignal.update((data) => {
       return {
         ...data,
-        'URL parameters': UrlParameters,
+        'URL parameters': {
+          ...window.toddle.locationSignal.get().params,
+          ...window.toddle.locationSignal.get().query,
```

Comment:
I believe we prefer path parameters over query parameters in our routing logic in case of naming conflict?
```suggestion
          ...window.toddle.locationSignal.get().query,
          ...window.toddle.locationSignal.get().params,

```